app/src/main/res/raw/initial_experiments.json

```diff
@@ -69,6 +69,64 @@
       "proposedEnrollment": 24,
       "referenceBranch": "control",
       "featureValidationOptOut": false
+    },
+    {
+      "schemaVersion": "1.10.0",
+      "slug": "test-android-notifications",
+      "id": "test-android-notifications",
+      "arguments": {},
+      "application": "org.mozilla.firefox_beta",
+      "appName": "fenix",
+      "appId": "org.mozilla.firefox_beta",
+      "channel": "beta",
```

Comment:
@jeddai we need some 👀 here, please. We published two first-run experiments for this featureID today, one for release and one for beta. However, I only see the beta one here? Am I missing it, or did something go wrong?

Looks like both are correctly returned from the API, but not in this file? 🤔 

/cc @rocketsroger

Reply:
Maybe it's also just cached and I am too impatient? :)

Reply:
Hm I think it might be an issue with the job... I'm going to close this, delete the branch and re-run the job

Reply:
Thank you, for looking into it!